push/src/main/java/org/aerogear/mobile/push/fcm/AeroGearFirebaseInstanceIdService.java

```diff
@@ -0,0 +1,6 @@
+package org.aerogear.mobile.push.fcm;
+
+import com.google.firebase.iid.FirebaseInstanceIdService;
+
+public class AeroGearFirebaseInstanceIdService extends FirebaseInstanceIdService {
```

Comment:
I forget, is there supposed to be a body in this class?

Reply:
Yes, I'm going to send a new PR about refresh token to not make it so, big and complicated!